app/views/users/profile_edit/_favorite_organization.html.erb

```diff
@@ -0,0 +1,15 @@
+<% org_type = user_fav_org.organization.org_type %>
+<div class="favorite-organization fav-org">
```

Comment:
why have `favorite-organization` and `fav-org`?

Reply:
Mostly just for flexibility.

Reply:
Flexibility??? what do you mean?

Reply:
Some people might like the whole thing being spelled out. Others may prefer the short form.

Reply:
let's just go with one I personally think it creates more confusion to implement something that's not needed

Reply:
no problem.